packages/components/src/MultiSelect/_docs/MultiSelect.mdx

```diff
@@ -12,7 +12,10 @@ import * as MultiSelectStories from "./MultiSelect.stories"
   className="!mb-8"
 />
 
-<KaioNotification />
+<SbReleaseNotification
+  type="negative"
+  content={<span>This component has not been officially released. Please wait for the an announcement in <code>#update_design_system</code> and for the design guidance to be published to Figma and Confluence before consuming.</span>}
```

Comment:
Ohhh. I meant bake these two into the component itself, not just rename the generic one 😂